SX1276/SX1276_LoRaRadio.cpp

```diff
@@ -718,25 +718,18 @@ uint32_t SX1276_LoRaRadio::time_on_air(radio_modems_t modem, uint8_t pkt_len)
             }
 
             // Symbol rate : time for one symbol (secs)
-            float rs = bw / (1 << _rf_settings.lora.datarate);
-            float ts = 1 / rs;
-            // time of preamble
-            float tPreamble = (_rf_settings.lora.preamble_len + 4.25f) * ts;
+            uint32_t tPreamble = (uint32_t) (((uint64_t) (((_rf_settings.lora.preamble_len * 1000) + 4250) * (1 << _rf_settings.lora.datarate))) / bw); // unit = ms
```

Comment:
There are extra parenthesis which makes the computation harder to read.
This is the same with less parenthesis:
```
(uint32_t) ( (uint64_t) ((_rf_settings.lora.preamble_len * 1000 + 4250) * (1 << _rf_settings.lora.datarate)) / bw)
```